t/t2260-job-list.t

```diff
@@ -797,6 +797,37 @@ test_expect_success HAVE_JQ 'flux job list lists nnodes for pending jobs correct
         flux queue start
 '
 
+test_expect_success 'reload the job-list module' '
```

Comment:
Just a small thought: The commit message for this change just says tests aren't "where they should be", but it would be nice to know _why_ the tests should be in a different place, not just that they are in the wrong place.